### DISCUSSION TITLE
More convenient hosting

### DISCUSSION THREAD

**Discussion by Tortik361** [FakeFishGames/Barotrauma#4787] [Ideas and suggestions]
Add the ability to limit the number of professions or add the ability to approve a profession from the host, (often ruiners take security officers, and they have access to the entire submarine, and then spoil the game, and it would be convenient if the host could only allow certain people to take a profession of a certain type) (sorry for my English)

**Comment by JamesTCrenshaw** [comment]
You may want to rename this topic to something like "Server host jobs approval" or something to that effect.
